lib/bolt.rb

```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
 
 module Bolt
-  require 'bolt/executor'
```

Comment:
This is probably not a good idea... maybe require something else?  Or find a way to make the executor load cheaper?